### PR TITLE
3057 remove pedistrian signal histogram admin

### LINKED ISSUE #3057: Remove Pedestrian Signal severity histogram from Admin page
##### Brief description of problem/feature
We added a histogram for severity ratings in Pedestrian Signal labels, but we removed severity from that label type shortly after. We just need to remove that histogram from the admin page now too!

Screenshot from 2022-10-21 11-53-06

### PR DESCRIPTION
**Pull request by sneh721**
Resolves #3057

Removed Pedestrian signal severity histogram from admin page, along with the corresponding average and standard deviation. Resized the remaining histograms. 

Before:
image

After:
image

##### Things to check before submitting the PR 

- [x] I've written a descriptive PR title. 
- [x] I've added/updated comments for large or confusing blocks of code.
- [x] I've included before/after screenshots above.

### CODE REVIEW

File: public/javascripts/Admin/src/Admin.js [hunk 1]
```diff
@@ -355,17 +355,13 @@ function Admin(_, $) {
                 $("#crosswalk-mean").html((crosswalkStats.mean).toFixed(2));
                 $("#crosswalk-std").html((crosswalkStats.std).toFixed(2));
 
-                var pedestrianSignalStats = getSummaryStats(pedestrianSignals, "severity");
```
Comment (misaugstad) [inline public/javascripts/Admin/src/Admin.js:358]: This line actually uses a variable called `pedestrianSignals` that is created above this section and needs to be deleted. Any time you're deleting something, make sure to check any variables/functions/HTML/CSS used in the deleted code, following it back to the source and deleting (after checking that it isn't used anywhere else)!

**Review by misaugstad — CHANGES_REQUESTED** [review]
Looks good, just one comment to address!

File: public/javascripts/Admin/src/Admin.js (hunk 1 above)
Comment (sneh721) [reply public/javascripts/Admin/src/Admin.js:358]: Sorry I missed that. I have removed the variable and pushed.

**Review by misaugstad — APPROVED** [review]
Looks great, thanks @sneh721 !

**Pull request merged**